app/modules/ads/index.js

```diff
@@ -332,36 +331,19 @@ class Ads {
   triggerAfterPageRenderServices() {
     const { bidders } = window.Wikia.adBidders;
     const { context, slotService } = window.Wikia.adEngine;
+    const { krux, moatYi, nielsen } = window.Wikia.adServices;
+    const targeting = context.get('targeting');
 
-    if (this.isAdStackEnabled()) {
-      biddersDelayer.resetPromise();
-      bidders.requestBids({
-        responseListener: biddersDelayer.markAsReady,
-      });
-      this.startAdEngine();
-
-      if (!slotService.getState('top_leaderboard')) {
-        this.finishFirstCall();
-      }
-    } else if (context.get('services.browsi.enabled')) {
-      // Browsi needs googletag loaded
-      this.loadGoogleTag();
-    }
```

Comment:
You can remove only this part of code and leave the rest of 336:364 as it is - in general this refactor with extracting `callExternalTrackingServices` function and checking if ad stack is enabled are ok and still valid so can stay.